subsys/net/lib/nrf_cloud/src/nrf_cloud_jwt.c

```diff
@@ -56,6 +69,13 @@ int nrf_cloud_jwt_generate(uint32_t time_valid_s, char *const jwt_buf, size_t jw
 		return err;
 	}
 
+	if (!modem_time_is_valid) {
+		int ret = ETIME;
+
+		LOG_WRN("Generated JWT has no expiration; modem does not have valid date/time");
+		return ret;
```

Comment:
I guess we could remove `ret` variable and just return `ETIME` here.

Reply:
I had to do this to avoid a CI warning